.kokoro/build.sh

```diff
@@ -46,6 +46,7 @@ export SENDGRID_API_KEY=$(cat $KOKORO_GFILE_DIR/secrets-sendgrid-api-key.txt)
 export FUNCTIONS_TOPIC=integration-tests-instance
 export FUNCTIONS_BUCKET=$GOOGLE_CLOUD_PROJECT
 export FUNCTIONS_DELETABLE_BUCKET=$GOOGLE_CLOUD_PROJECT-functions
+export BASE_URL="https://$GCF_REGION-$GOOGLE_CLOUD_PROJECT.cloudfunctions.net/"
```

Comment:
I believe this change is breaking the `functions/speech-to-speech tests because of this line: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/220c3f4a90bb06c434c9886f3c987148272c4462/functions/speech-to-speech/functions/test/index.test.js#L29

Reply:
Thank you for helping me debug this!!